### ISSUE TITLE
Cannot re-request location

### ISSUE THREAD

**Issue by nessthehero** [senchalabs/jQTouch#338]
I want the user to be able to click a button and they will be asked for their location information again, so that if they were say, moving down a street or something, they could reset their location and then if I was grabbing data based on Lat and Lon, it would reflect the new location.

Currently, I have the location being requested in an earlier method, but I want them to be able to navigate to a menu and click a button to re-request the location.

Clicking the button does nothing.

I also notice that if they say "No" to the request, there is literally no way to tell. Any ideas? I am setting the jqt.updateLocation method to a lookup variable, but the console returns that lookup variable as "true" whether or not I approve the permissions.

TL;DR: I want them to click a button and ask them for their location again, and again. Every time they click the button.
